api/v1alpha1/loadbalancer_types.go

```diff
@@ -134,6 +144,45 @@ type Cookie struct {
 	Attributes map[string]string `json:"attributes,omitempty"`
 }
 
+// ClientSideWeightedRoundRobin defines configuration for Envoy's Client-Side Weighted Round Robin policy.
+// See Envoy proto: envoy.extensions.load_balancing_policies.client_side_weighted_round_robin.v3.ClientSideWeightedRoundRobin
+// Note: SlowStart is not supported for this policy in Envoy Gateway at this time.
+type ClientSideWeightedRoundRobin struct {
```

Comment:
in the first iteration, do we need to add all these fields, or can be rely on the defaults that are preset ?
can you share the motivation for adding these fields

Reply:
So I have removed the other fields which are not currently supported by envoy, but at present I guess the fields being published are important to properly configure the system.

For example black out period can be different for different applications based on how much time they take to get the metrics state. We have some apps which does heavy data loading in the beginning and hence their blackout periods are high.

I think all of these metrics are application specific and we have in our sytems already configured / tuned them for different services. So since the implementation is already done are there any concerns of why we should not include them?